apps/discovery-platform/src/db/index.ts

```diff
@@ -38,15 +38,15 @@ export const getRegisteredNodes = async (
   log.verbose("Querying for Registered nodes with filters", filters);
   let baseText = `SELECT * FROM ${TABLES.REGISTERED_NODES}`;
   let filtersText = [];
-  const values: { [key: string]: string } = {};
+  const values: { [key: string]: string | string[] } = {};
 
   if (filters?.excludeList !== undefined && filters.excludeList.length !== 0) {
-    filtersText.push("id NOT IN ($<list>)");
-    values["list"] = filters.excludeList.join(", ");
+    filtersText.push("id NOT IN ($<list:csv>)");
```

Comment:
does `:` mean **OR**?
might be good to document it here

Reply:
documented, it actually is a formatting filter that gets a Javascript array and turns it into a list of Comma-Separated Values.